src/components/shared/testimonial/testimonial.jsx

```diff
@@ -1,9 +1,12 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+
 import Container from 'components/shared/container/container';
 
 import QuoteIcon from './images/quote.inline.svg';
 
-const Testimonial = () => (
-  <figure className="testimonial safe-paddings mt-[200px] 2xl:mt-40 xl:mt-36 lg:mt-28 md:mt-20">
+const Testimonial = ({ className = '', quote, name, position }) => (
+  <figure className={clsx('testimonial safe-paddings', className)}>
```

Comment:
Let's swap the `figure` and the `Container` if possible, because `figcaption` not allowed as child of `div`